net-misc/youtube-viewer/youtube-viewer-3.5.8.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils gnome2-utils perl-module
+
+DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
+HOMEPAGE="https://trizenx.blogspot.com/2012/03/gtk-youtube-viewer.html"
+SRC_URI="https://github.com/trizen/youtube-viewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk"
+
+RDEPEND="
+	dev-perl/Data-Dump
+	dev-perl/JSON
+	dev-perl/libwww-perl[ssl]
+	dev-perl/Term-ReadLine-Gnu
+	dev-perl/LWP-Protocol-https
+	virtual/perl-Encode
+	virtual/perl-File-Path
+	virtual/perl-File-Spec
+	virtual/perl-Getopt-Long
+	virtual/perl-Scalar-List-Utils
+	virtual/perl-Term-ANSIColor
+	virtual/perl-Term-ReadLine
+	virtual/perl-Text-ParseWords
+	virtual/perl-Text-Tabs+Wrap
+	gtk? (
+		dev-perl/File-ShareDir
+		>=dev-perl/Gtk2-1.244.0
+		virtual/freedesktop-icon-theme
+		x11-libs/gdk-pixbuf:2[X,jpeg]
+	)
+	|| ( media-video/ffmpeg[openssl] media-video/ffmpeg[gnutls] )
```

Comment:
No love for libressl? :)
(Im not asking you to test it, just if you are aware whether it works or not)

Reply:
I've never tried libressl, I'll need to set up a Gentoo VM with libressl in it for this to be tested. I'll check, but resolving this issue might be out of scope for this PR.

Reply:
I agree. As I said, I dont expect you to set up a libressl system for this one. Just if you were aware that it works/not from upstream issues, changelogs etc.

Reply:
I'll check with upstream or will test it myself. I'll open a bug to myself in bugzilla to track this.